pages/team.html

```diff
@@ -0,0 +1,143 @@
+<script>
+	{
+		"title": "The jQuery Team",
+		"customFields": [
+			{
+				"key": "subtitle",
+				"value": "The jQuery Project is run by a distributed group of volunteers."
+			},{
+				"key": "full-width",
+				"value": 1
+			}
+		]
+	}
+</script>
+
+<h1 class="block">The jQuery Team</h1>
+
+<ul class="block-grid five-up mobile-two-up team-list">
+	<li>
+		<img src="https://secure.gravatar.com/avatar/f16d4602c1c90646438a0b534ff61889?size=120" alt="">
+		<h2>Richard Gibson</h2>
+		<h3>Sizzle Lead | Core | QUnit</h3>
+	</li>
+	<li>
+		<img src="https://secure.gravatar.com/avatar/72ea0622f7b690ed82f5261b16ff8cef?size=120" alt="">
+		<h2>Michał Gołębiowski</h2>
+		<h3>Core</h3>
+	</li>
+	<li>
+		<img src="https://secure.gravatar.com/avatar/49061e8503ca321a8045c9065963a8c6?size=120" alt="">
+		<h2>Timo Tijhof</h2>
+		<h3>QUnit | Infrastructure Lead</h3>
+	</li>
+	<li>
+		<img src="https://secure.gravatar.com/avatar/6b57220ae48e1ac5d02227bdada8a4d6?size=120" alt="">
+		<h2>Timmy Willison</h2>
+		<h3>Core Lead</h3>
+	</li>
+	<li>
+		<img src="https://secure.gravatar.com/avatar/6e836e35b57421faa0067702089ed46e?size=120" alt="">
+		<h2>Rafael Xavier</h2>
+		<h3>Globalize Lead</h3>
+	</li>
```

Comment:
Do we still consider Globalize as part of the jQuery portfolio? I know there were plans to rebuild some jQuery UI widgets on top of globalize but I'm not sure if anything really happened here (@fnagel?).

Reply:
I don't have a preference either way.

I figured for jQuery Core/UI/Mobile/Infra it's easiest to keep a single team page here. Globalize does use jQuery Infra for its domain redirect, and its branding/logo follow the jQuery style. QUnit likewise, but QUnit has its own website and team. 

For Globalize we don't have a website, and it's not deprecated or archived (essentially a direct project under OpenJS). We could also list Rafael as former jQuery Team member, "Globalize Lead" but it's slightly confusing in that state since it is a current project and current role. I left Rafael as current team member since that's how it is today and seems the least change in one go 🙂. I'll follow what you and Timmy prefer.

Reply:
Globalize may not be deprecated or archived, but it's had very little activity since 2018. Perhaps it should be deprecated?

Reply:
Yes, the former jQuery UI team planned to integrate it into the reworked calendar / datepicker widgets. Sadly those have never been released but Globalize was utilized for localization of the calendar widget. See here for more info:

https://jqueryui.pbworks.com/w/page/12137778/Datepicker
https://github.com/jquery/jquery-ui/tree/datepicker

Afaik Globalize is no longer active. It died more or less when Scott, Jörn and all the others left the team.
When looking at the commits, its only kept alive but not actively developed.

Reply:
I asked on the OpenJSF Slack how do we want to approach Globalize.

@rxaviers any input from you would be very helpful here as well!